File: atm-8/ATM.Domain/Rule.cs
using ATM.Logic;
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using System.Threading.Tasks;

namespace ATM.Domain
{
    public class Rule
    {
        public ClauseOperators Operator { get; set; }
        public List<object> Conditions { get; set; }
        public RuleOperator RuleOperator { get; set; }
        public List<object> Actions { get; set; }
    }
}

File: atm-6/java/Rule.java
import java.util.List;

public class Rule {
	public RuleOperator op;
	public List<Object> conditions;
	public List<Object> consequences;

	public Rule(RuleOperator op, List<Object> conditions, List<Object> consequences) {
		this.op = op;
		this.conditions = conditions;
		this.consequences = consequences;
	}
}